tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -129,7 +128,7 @@ def test_user_count(self):
             self.url,
             data={
                 "start": iso_format(self.day_ago),
-                "end": iso_format(self.day_ago + timedelta(hours=1, minutes=59)),
+                "end": iso_format(self.day_ago + timedelta(hours=2)),
```

Comment:
@iProgramStuff why were these `timedelta()` updated?

Reply:
I did a replace on all the timedelta's using 1 hour and 59 minutes to using 2 hours. Some tests were failing because it wasn't including the last bucket because we were no longer adding the rollup.